furnace.c

```diff
@@ -465,12 +529,33 @@ format_status(char* buffer, furnace_context_t* ctx)
       MAX_PWM,
       ctx->pilot.is_enabled
     );
+#endif
+}
+
+#if CONFIG_AUTO == CONFIG_AUTO_MAPPER
+
+static int
+format_mapper(char *buffer, furnace_context_t *ctx)
+{
+    return snprintf(
+      buffer,
+      MAPPER_STATUS_SIZE,
+      MAPPER_STATUS_FMT,
+      ctx->pwm_level,
+      ctx->mapper.max_pwm_temp
+    );
 }
 
+#endif
```

Comment:
Why don't we reuse `format_status()`? I guess It's doing the same thing.

Reply:
It's not doing the same thing. format_status is used like always, printing every second, when format_mapper is used only, when changing to next pwm to signalize the max temperature.

Reply:
Ok. Makes sense.